src/wp-content/themes/twentyseventeen/inc/template-tags.php

```diff
@@ -140,7 +140,7 @@ function twentyseventeen_edit_link() {
  * @param int                  $id      Front page section to display.
  */
 function twentyseventeen_front_page_section( $partial = null, $id = 0 ) {
-	if ( is_a( $partial, 'WP_Customize_Partial' ) ) {
+	if ( $partial instanceof \WP_Customize_Partial ) {
```

Comment:
I think we shouldn't do the modifications for the bundled themes in this ticket.

Reply:
May I ask why that is? It is part of WordPress core, and because this is not an API change, I think changing parts of core that people might extend or reuse is still fine.

If the changes were to be merged with changes in the themes, that makes we have zero `is_a` calls.

Reply:
@Ayesh Themes often support a wider range of WP versions/PHP versions, so we need to be wary of making changes to them which could cause cross-version compatibility issues, though I don't see how this change would cause such an issue as `instanceof` is available with the above syntax since PHP 5.0.

Reply:
Yes, apart from that, also I wanted to keep this ticket for only the changes in the core, we can do these changes in another ticket for the Bundled Themes using the Bundled Themes component, for help in tracking.

Reply:
FWIW, bundled theme files are often included in general optimizations like this, so I think it's fine to keep this change here, no need for a separate PR 👍